types/options.d.ts

```diff
@@ -88,7 +88,7 @@ export interface EllipseAnnotationOptions extends CoreAnnotationOptions, Annotat
   rotation?: Scriptable<number, PartialEventContext>
 }
 
-export interface PointAnnotationOptions extends CoreAnnotationOptions, AnnotationPointCoordinates {
+export interface PointAnnotationOptions extends CoreAnnotationOptions, AnnotationCoordinates, AnnotationPointCoordinates {
```

Comment:
Should the coordinate types be merged?

Reply:
Do you mean in a unique interface? No, because box and ellipse are using only x/yMin and x/yMax and not x/yValue, bu I could extend PointCoordinates from Coordinates. Ok for you?

Reply:
Was just a thought, ok to leave as is.

Reply:
have a look! I have change a bit with extends, therefore you don't have some many extends in the annotation options